processing/src/main/java/io/druid/query/search/SearchQueryQueryToolChest.java

```diff
@@ -193,7 +208,9 @@ public SearchQueryQueryToolChest(
           @Override
           public Object apply(Result<SearchResultValue> input)
           {
-            return Lists.newArrayList(input.getTimestamp().getMillis(), input.getValue());
+            return dimensionSpecs != null
```

Comment:
agree that `dimensionSpecs` as written will never be null, so this conditional is unnecessary.